src/main/java/com/hpe/simulap/protocol/sip/sampler/SipSampler.java

```diff
@@ -1655,7 +1657,7 @@ private void receiveAnswerMessage(SampleResult res) {
 
 
             if (respQueue == null) {
-            	if (_logger.isDebugEnabled()) _logger.debug("resp queue not created for id "+id +".",null);				
+		if (_logger.isDebugEnabled()) _logger.debug("resp queue not created for id "+id +".","");
```

Comment:
debug method could be rewritten here like this if (_logger.isDebugEnabled()) _logger.debug("resp queue not created for id {}., id);